eng/mgmt/changelog/src/main/java/com/azure/resourcemanager/tools/changelog/changelog/ChangeLog.java

```diff
@@ -0,0 +1,139 @@
+        Map<String, AllMethods> allMethods = new HashMap<>();
+        AllMethods.fromClasses(classMap, allMethods);
+        Map<String, Map<String, AllMethods>> stages = new HashMap<>();
+        List<ChangeLog> changeLogForNonStage = allMethods.entrySet().stream().map(entry -> {
+            String namespace = ClassName.namespace(entry.getKey());
+            String parentClass = ClassName.parentName(entry.getKey());
+            String parentName = ClassName.name(parentClass);
+            if (parentName.equals("DefinitionStages")) {
+                stages.computeIfAbsent(namespace + "." + parentClass, key -> new HashMap<>()).put(entry.getKey(), entry.getValue());
+            } else if (!parentName.equals("UpdateStages")) {
+                return new ChangeLog(entry.getValue());
+            }
+            return null;
+        }).filter(Objects::nonNull).collect(Collectors.toList());
+
+
+        Stream<ChangeLog> changeLogForStage = stages.entrySet().stream().map(entry -> new DefinitionStageChangeLog(entry.getValue(), entry.getKey()));
+        return Stream.concat(changeLogForStage, changeLogForNonStage.stream()).collect(Collectors.toList());
+    }
+
+    public JApiClass getJApiClass() {
+        return this.allMethods.getJApiClass();
+    }
+
+    public String getNewFeature() {
+        StringBuilder builder = new StringBuilder();
+        for (int i = 0; i < this.newFeature.size(); ++i) {
+            builder.append(this.newFeature.get(i)).append("\n");
+            if (i + 1 == this.newFeature.size()) {
+                builder.append("\n");
+            }
+        }
+        return builder.toString();
+    }
+
+    public String getBreakingChange() {
+        StringBuilder builder = new StringBuilder();
+        for (int i = 0; i < this.breakingChange.size(); ++i) {
+            builder.append(this.breakingChange.get(i)).append("\n");
+            if (i + 1 == this.breakingChange.size()) {
+                builder.append("\n");
+            }
+        }
+        return builder.toString();
+    }
+
+    public boolean isClassLevelChanged() {
+        return getJApiClass().getChangeStatus() == JApiChangeStatus.NEW || getJApiClass().getChangeStatus() == JApiChangeStatus.REMOVED;
+    }
+
+    protected void calcChangeLog() {
+        calcChangeLogForClass();
+    }
+
+    private void calcChangeLogForClass() {
+        switch (getJApiClass().getChangeStatus()) {
+            case NEW: newFeature.add(String.format("* `%s` was added", getJApiClass().getFullyQualifiedName())); break;
+            case REMOVED: breakingChange.add(String.format("* `%s` was removed", getJApiClass().getFullyQualifiedName())); break;
+            default:
+                AtomicBoolean checkReturnType = new AtomicBoolean(true);
+                if (ClassName.name(getJApiClass()).equals("Definition")) {
+                    checkReturnType.set(false);
+                }
```

Comment:
Easier to use `? :` (or just another variable name as constant) for constant variable, instead of `AtmoicBoolean`.